modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/registry/ObjectServiceUpgradeStepRegistrator.java

```diff
@@ -175,7 +175,12 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 			new com.liferay.object.internal.upgrade.v3_18_0.
 				ObjectFieldUpgradeProcess(_portalUUID));
 
-		registry.register("3.18.0", "3.19.0", ObjectFilterTable.create());
+		registry.register(
+			"3.18.0", "3.18.1",
+			new com.liferay.object.internal.upgrade.v3_18_1.
+				ObjectViewColumnUpgradeProcess());
```

Comment:
@jvelalr can we create a new schema version, after the current latest schema version, instead of adding between two existing upgrades? In this case it would be after the:

```
registry.register(
		"4.0.0", "4.1.0", new ObjectDefinitionUpgradeProcess());
```

This way, we can ensure that all customers will execute this new upgrade.